### PR TITLE
Added new CSS to avoid using card ID

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — COMMENTED** [review]
@JV0812 thank you SO MUCH for figuring this out!! You are the best. Are all index.md pages fixed in this PR, or just some? The reason I ask is that it'll break any index.md pages that still have "smallbox[number]".

**Comment by JV0812** [comment]
@kimsauce I have updated all of the available index pages.

**Comment by jpipkin1** [comment]
@JV0812, the build-and-deploy failed ([Details](https://github.com/SumoLogic/sumologic-documentation/actions/runs/6470333725/job/17566299200?pr=3179)). Could you fix it? Then I'll give this PR a review.

File: docs/integrations/web-servers/index.md [hunk 1]
```diff
@@ -10,78 +10,78 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 This guide has documentation for Sumo apps for web servers.
 
 
-<div className="box-wrapper" markdown="1">
-<div className="box smallbox1 card">
+<div className="box-wrapper">
+<div className="box smallbo card">
```
Comment (jpipkin1) [inline docs/integrations/web-servers/index.md:14]:
```suggestion
<div className="box smallbox card">
```

File: docs/integrations/web-servers/index.md [hunk 2]
```diff
@@ -10,78 +10,78 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
   <img src={useBaseUrl('img/integrations/web-servers/squid-proxy.png')} alt="Thumbnail icon" width="50"/>
   <h4><a href="/docs/integrations/web-servers/squid-proxy">Squid Proxy</a></h4>
   <p>The Squid Proxy app is a unified logs and metrics app that helps you monitor activity in Squid Proxy.</p>
   </div>
 </div>
-<div className="box smallbox11 card">
+<div className="box smallbox card">
   <div className="container">
   <img src={useBaseUrl('img/integrations/web-servers/varnish-cache.png')} alt="Thumbnail icon" width="190"/>
   <h4><a href="/docs/integrations/web-servers/varnish">Varnish</a></h4>
```
Comment (jpipkin1) [inline docs/integrations/web-servers/index.md:87]: Below change "box smallbox12 card" to "box smallbox card".

**Review by jpipkin1 — APPROVED** [review]
Awesome work, @JV0812! I found very few things to fix. I approve the PR contingent on your making the updates.

**Comment by kimsauce** [comment]
> @kimsauce I have updated all of the available index pages.

Thank you for clarifying! It looks like the last outstanding item is a minor merge conflict on one page.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@JV0812, are you ready to merge this PR now? I'm excited to see this one in production!

**Comment by JV0812** [comment]
Yes John, I will merge it today.

**Pull request merged**